phoenixRest/models/core/agenda_entry.py

```diff
@@ -30,22 +30,52 @@ class AgendaEntry(Base):
     event_uuid = Column(UUID(as_uuid=True), ForeignKey("event.uuid"), nullable=False)
     event = relationship("Event")
     
+    title = Column(Text, nullable=False)
+    description = Column(Text, nullable=True, server_default=None)
+    location = Column(Text, nullable=True, server_default=None)
     time = Column(DateTime, nullable=False)
 
-    title = Column(Text, nullable=False)
-    description = Column(Text, nullable=False)
+    deviating_time = Column(DateTime, nullable=True, server_default=None)
+    deviating_location = Column(Text, nullable=True, server_default=None)
+    deviating_information = Column(Text, nullable=True, server_default=None)
+
+    state_pinned = Column(Boolean, server_default='false')
+    state_deviating_time_unknown = Column(Boolean, server_default='false')
+    state_cancelled = Column(Boolean, server_default='false')
+
+    log_created_by_uuid = Column(UUID(as_uuid=True), ForeignKey("user.uuid"), nullable=False)
+    log_created_by_time = Column(DateTime, nullable=False, server_default='NOW()')
+    log_modified_by_uuid = Column(UUID(as_uuid=True), ForeignKey("user.uuid"), nullable=True, server_default=None)
+    log_modified_by_time = Column(DateTime, nullable=True, server_default=None)
 
-    def __init__(self, title: str, description: str, time: datetime, event: Event):
+    def __init__(self, event: Event, title: str, description: str, location: str, time: DateTime, state_pinned: bool, log_created_by_uuid: str):
```

Comment:
by passing in UUIDs you are not using sqlalchemy properly. You should pass in `User` objects and use the `relationship()` function.